edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfigProvider.cs

```diff
@@ -62,7 +76,11 @@ CombinedDockerConfig GetCombinedConfig(IModule module, IRuntimeInfo runtimeInfo)
             // if the workload URI is a Unix domain socket then volume mount it into the container
             CreateContainerParameters createOptions = CloneOrCreateParams(combinedConfig.CreateOptions);
             this.MountSockets(module, createOptions);
-            this.InjectNetworkAliases(module, createOptions);
+
+            if (!string.IsNullOrWhiteSpace(this.networkId))
```

Comment:
What if the id is null/empty with the extention feature enabled?

Reply:
Network aliases is an optional thing here and will be just ignored. @darobs do we use this when generating deployment?

Reply:
These network aliases we're inserting into the Docker Config aren't used by the docker->pod translation. Calls to `InjectNetworkAliases` should probably be removed to avoid this confusion.

Reply:
Oh, that's nice :)

Reply:
Deleted